### ISSUE TITLE
[Datastore Management] Include Icon with datastore connection (FE)

### ISSUE THREAD

**Issue by adriaaaa** [ethyca/fidesops#671]
### Is your feature request related to a specific problem?

As part of the datastore management [landing page](https://github.com/ethyca/fidesops/issues/601), we will display a list of datastores that have been connected. To help users easily manage the list, we will be including the logos of the type of connector. We will ultimately need a more streamlined solution for including logos (and allowing end users to upload logos for their custom connectors that they build) but for now we just need a way to display the proper logo.

### Describe the solution you'd like
-display the default logo (see attached figma file) for all connected datastores
-make sure to error gracefully if it cannot find an image (can use default icon)
-add info to connector that will help describe type and then serve svg (optional/nice to have)

### Additional context

[Default logo](https://www.figma.com/file/MTbZohIGvB4bq2f2wkiDHO/Fidesops-Management-UI?node-id=1542%3A17489)

**Comment by adriaaaa** [comment]
Hey @eastandwestwind sorry for the confusion, I know there was a lot of back and forth on this one in the last week. 

1. That is correct although we said that app-specific logos were a nice to have. At the minimum we should display that ethyca thumbprint logo for each connected datastore. If it's possible for us to get the connectors to a point where we can, for example, display the mailchimp logo on the connected mailchimp connector within this sprint, then that is great. I'll send you the SVGs for each of our supported connectors. We can consider custom logos (like if someone wrote their own connector and wanted to upload an emoji as the logo or something) out of scope.
2. I think my answer to number one above might have answered part of this question? And yes, if a specific app's logo is not found, we can display the ethyca thumbprint logo as default.

**Comment by eastandwestwind** [comment]
So to clarify, those 10 supported connector SVGs you sent (Postgres, Sqlserver, hubspot, mongodb, redshift, snowflake, mailchimp, mariadb, mysql, segment) includes **both** SaaS and databases.

So instead of thinking of logo functionality as SasS vs database, we can alter acceptance criteria to something like this:

1. Display the datastore's logo if we support the specific datastore (includes 10 across SaaS and db for now, but will grow in future)
2. Otherwise, just display default logo

Does that sound better?

**Comment by adriaaaa** [comment]
> So to clarify, those 10 supported connector SVGs you sent (Postgres, Sqlserver, hubspot, mongodb, redshift, snowflake, mailchimp, mariadb, mysql, segment) includes both SaaS and databases.
Yep

> Does that sound better?

sounds good! @eastandwestwind

**Comment by chriscalhoun1974** [comment]
Currently, I am waiting on new .svg files from Simon for the Connection Logos. In addition, I need to know the new schema field to determine the Sass type on `api/v1/connection` API endpoint. This was discussed earlier this week during standup.
